cmd/kaniko-docker/main.go

```diff
@@ -61,6 +61,11 @@ func main() {
 			EnvVar:   "PLUGIN_TAGS",
 			FilePath: ".tags",
 		},
+		cli.BoolFlag{
```

Comment:
Add this flag for kaniko-gcr & kaniko-ecr as well.

Reply:
Done